opencti-platform/opencti-front/src/private/components/settings/sub_types/SubType.tsx

```diff
@@ -60,43 +64,50 @@ const SubType = ({ data }: { data: SubType_subType$key }) => {
   const classes = useStyles();
   const subType = useFragment(subTypeFragment, data);
   const statuses = (subType.statuses?.edges ?? []).map((edge) => edge.node);
-  const queryRef = useQueryLoading<EntitySettingQuery>(entitySettingQuery, {
-    targetType: subType.id,
-  });
+
+  const subTypeSettingsId = subType.settings?.id;
+  if (subTypeSettingsId) {
+    const config = useMemo<GraphQLSubscriptionConfig<SubTypeEntitySettingSubscription>>(
```

Comment:
Yes, we saw this together, and I think it's because we use a state in EntitySettingScale. I don't know how I could reset the state when scaleConfig changes. Is there a method to allow us to know when there is a change ? I'm searching about it.

Reply:
I investigated and didn't find a solution for now